ArduPlane/GCS_Mavlink.cpp

```diff
@@ -791,13 +792,197 @@ MAV_RESULT GCS_MAVLINK_Plane::handle_command_int_do_reposition(const mavlink_com
     return MAV_RESULT_FAILED;
 }
 
+// these are GUIDED mode commands that are RATE or slew enabled, so you can have more powerful control than default controls.
+MAV_RESULT GCS_MAVLINK_Plane::handle_command_int_guided_slew_commands(const mavlink_command_int_t &packet)
+{
+
+
+  ::printf("GCS_MAVLINK_Plane::handle_command_int_guided_slew_commands .. slew packet recieved:%d\n",packet.command);
+
+  switch(packet.command) {
+    
+
+#if OFFBOARD_GUIDED == ENABLED
+    case MAV_CMD_GUIDED_CHANGE_SPEED: {
+        // command is only valid in guided
+        if (plane.control_mode != &plane.mode_guided) {
+            gcs().send_text(MAV_SEVERITY_WARNING,"TEMPORARILY_REJECTED - not in guided mode.");
```

Comment:
All these send_texts need to go away, these are debug ones, and the MAV_RESULT is the handler for this.

Reply:
will fix on next attempt.